src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -4546,6 +4546,132 @@ default Emote getEmoteById(long id)
     @CheckReturnValue
     AuditableRestAction<Void> unban(@Nonnull String userId);
 
+    /**
+     * Puts the specified Member in time out in this {@link net.dv8tion.jda.api.entities.Guild Guild} until the specified date.
+     * <br>While a Member is in time out, they cannot reply, send messages, react and speak in voice channels.
+     *
+     * <p>Possible {@link net.dv8tion.jda.api.requests.ErrorResponse ErrorResponses} caused by
+     * the returned {@link net.dv8tion.jda.api.requests.RestAction RestAction} include the following:
+     * <ul>
+     *     <li>{@link net.dv8tion.jda.api.requests.ErrorResponse#MISSING_PERMISSIONS MISSING_PERMISSIONS}
+     *     <br>The target Member cannot be timed out due to a permission discrepancy</li>
+     *
+     *     <li>{@link net.dv8tion.jda.api.requests.ErrorResponse#UNKNOWN_MEMBER UNKNOWN_MEMBER}
+     *     <br>The specified Member was removed from the Guild before finishing the task</li>
+     * </ul>
+     *
+     * @param  member
+     *         The member to put in time out
+     * @param  date
+     *         The date until which the specified Member should be in time out
+     *
+     * @throws net.dv8tion.jda.api.exceptions.InsufficientPermissionException
+     *         If the logged in account does not have the {@link net.dv8tion.jda.api.Permission#MODERATE_MEMBERS} permission.
+     * @throws net.dv8tion.jda.api.exceptions.HierarchyException
+     *         If the logged in account cannot put a timeout on the other Member due to permission hierarchy position.
+     *         <br>See {@link Member#canInteract(Member)}
+     * @throws IllegalArgumentException
+     *         If any of the following checks fail
+     *         <ul>
+     *             <li>The provided {@code member} is not null</li>
+     *             <li>The provided {@code date} is not null</li>
+     *             <li>The provided {@code date} is not in the past</li>
+     *             <li>The provided {@code date} is not more than 28 days in the future</li>
+     *         </ul>
```

Comment:
This wording is confusing IMO.
```suggestion
     *         If any of the following are true
     *         <ul>
     *             <li>The provided {@code member} is null</li>
     *             <li>The provided {@code date} is null</li>
     *             <li>The provided {@code date} is in the past</li>
     *             <li>The provided {@code date} is more than 28 days in the future</li>
     *         </ul>
```